### PR TITLE
Remove source repository info

### PR DESCRIPTION
**Pull request by andresmgot**
Ref: #53

### CODE REVIEW

File: src/app/chart-details/chart-details-info/chart-details-info.component.ts [hunk 1]
```diff
@@ -24,13 +24,6 @@ export class ChartDetailsInfoComponent implements OnInit {
     return this.chart.attributes.sources || [];
```
Comment (prydonius) [inline src/app/chart-details/chart-details-info/chart-details-info.component.ts]: what about this?

Comment (andresmgot) [reply src/app/chart-details/chart-details-info/chart-details-info.component.ts]: this is not the same, we are removing `chart.attributes.repo.source` not `chart.attributes.sources`. I don't know if we should remove that as well but you didn't remove that in the PR you point.

Comment (prydonius) [reply src/app/chart-details/chart-details-info/chart-details-info.component.ts]: Oops sorry, you're right

File: src/app/chart-details/chart-details-info/chart-details-info.component.ts [hunk 2]
```diff
@@ -11,8 +11,10 @@ import * as urljoin from 'url-join';
   styleUrls: ['./chart-details-info.component.scss']
 })
 export class ChartDetailsInfoComponent implements OnInit {
-  @Input() chart: Chart;
-  @Input() currentVersion: ChartVersion;
+  @Input()
```
Comment (prydonius) [inline src/app/chart-details/chart-details-info/chart-details-info.component.ts]: did your editor change this automatically or something?

Comment (andresmgot) [reply src/app/chart-details/chart-details-info/chart-details-info.component.ts]: yep, the auto-formatting thing

**Pull request merged**